modin/pandas/test/test_dataframe.py

```diff
@@ -2358,8 +2358,7 @@ def test_interpolate(self):
 
     def test_kurt(self):
         data = test_data_values[0]
-        with pytest.warns(UserWarning):
-            pd.DataFrame(data).kurt()
+        pd.DataFrame(data).kurt()
```

Comment:
is there any test at all?
maybe test function should look like this?
```python
    modin_df, pandas_df = pd.DataFrame(data), pandas.DataFrame(data)
    df.equals(modin_df.kurt(), pandas_df.kurt())
```

Reply:
fixed